### DISCUSSION TITLE
Is there a way to visualize heightfields?

### DISCUSSION THREAD

**Discussion by vihdutta** [jcornaz/heron#176] [Q&A] [answered]
I'm trying to add a heightfield for a 3D height-mesh. I have all the points from the 3D height-mesh formatted for a `CollisionShape::HeightField`, but I have some odd behavior and would like to see the heightfield. How can I do this? I am currently generating the collision heightfield similar to what is shown below:

```
    commands
        .spawn_bundle((Transform::identity(), GlobalTransform::identity()))
        .insert(RigidBody::Static)
        .insert(CollisionShape::HeightField {
            size: Vec2::new(10.0, 10.0),
            heights: vec![
                vec![1.5, 0.8, 0., 0., 3.0],
                vec![3., 3., 3., 3., 3.0],
            ],
        });
```

**Comment by jcornaz** [comment]
There was a 3d-debug rendering before. But It's been dropped when upgrading to bevy 0.6. This is because the rendering was using [bevy_debug_lines](https://github.com/toqozz/bevy_debug_lines) which hasn't yet be ported to bevy 0.6 yet.

Hopefuly, we'll be able to reintroduce eventually.

**Comment by jcornaz** [comment, marked as answer]
The 3d debug renderer has been reintroduced in version [2.2.0](https://github.com/jcornaz/heron/releases/tag/v2.2.0)
